recipes/flite/install.sh

```diff
@@ -0,0 +1,5 @@
+set -exou
+
+mkdir -p "${PREFIX}/lib"
+
+cp -P build/lib/lib*${SHLIB_EXT}.* "${PREFIX}/lib/"
```

Comment:
Maybe this fixes the macOS build.

```suggestion
cp -P build/lib/*${SHLIB_EXT}* "${PREFIX}/lib/"
```